solarforecastarbiter/reports/main.py

```diff
@@ -95,8 +96,15 @@ def get_data_for_report(session, report):
             data[fxobs.forecast] = session.get_values(
                 fxobs.forecast, start, end)
         if fxobs.data_object not in data:
-            data[fxobs.data_object] = session.get_values(
+            obs_data = session.get_values(
                 fxobs.data_object, start, end)
+            if isinstance(
+                    fxobs.data_object, datamodel.Aggregate):
+                data[fxobs.data_object] = obs_data
```

Comment:
I can't decide if I prefer this if statement here or within `apply_validation`. I think I'd lean towards the latter since it seems like it decouples the report and validation code.